### ISSUE TITLE
Calling a command with invalid arguments results in zero exit code

### ISSUE THREAD

**Issue by morozov** [c9s/CLIFramework#106]
Please see phpbrew/phpbrew/issues/822 for the reference. Internally, the following is happening:
```
↪ bin/phpbrew env php-7.0-whatever-invalid-value
Invalid argument php-7.0-whatever-invalid-value

↪ echo $?
0
```
It makes `.phpbrew/bashrc` execute this output by means of `eval` instead of breaking.

@c9s please create the `3.0` branch in this repository based off of df9139a (this is what PHPBrew currently uses). I'll rebase #107 then.

**Comment by c9s** [comment]
Ok i will do this


Sergei Morozov <notifications@github.com>於 2017年11月6日 週一，02:35寫道：

**Comment by morozov** [comment]
Fixed by #107.
